9.py

```diff
@@ -0,0 +1,46 @@
+
+class BankAccount:
+    def __init__(self, account_number, owner_name, balance):
+        self.account_number = account_number
+        self.owner_name = owner_name
+        self.balance = balance
```

Comment:
tutaj nie jest to błąd, ale czasem warto się zastanowić czy lepiej trzymać kwoty pieniężne w typie Decimal
np.
```py
from decimal import Decimal
print(Decimal("1.1")+Decimal("2.2"))
```